### DISCUSSION TITLE
3.3.0大家的123盘是否能正常列表？

### DISCUSSION THREAD

**Discussion by AspirantH** [AlistGo/alist#2188] [Q&A]
是不是123PAN接口变动了？
failed get objs: failed to list objs: json: cannot unmarshal string into Go struct field Files.code of type int

**Comment by AspirantH** [comment]
有小伙伴遇到吗？

**Reply by AspirantH** [reply]
不知道大家的123pan是否能正常列表

**Reply by xuchuang199636** [reply]
加一，有些目录无法正常显示列表。

**Reply by xzzxyz** [reply]
+1 报错和你一样 之前放了几个小文件测试还没问题 后来传了几个G图片就这样了

**Reply by fanfanqie** [reply]
请问要怎么解决啊- -打不开文件呢

**Comment by AspirantH** [comment]
[GIN] 2022/11/10 - 11:39:12 | 200 |     934.781µs |   192.168.1.210 | GET      "/123PAN"
[GIN] 2022/11/10 - 11:39:12 | 200 |     1.92969ms |   192.168.1.210 | GET      "/assets/index.659f4289.css"
[GIN] 2022/11/10 - 11:39:12 | 200 |    8.538994ms |   192.168.1.210 | GET      "/assets/index.403084e8.js"
[GIN] 2022/11/10 - 11:39:13 | 200 |    2.084603ms |   192.168.1.210 | GET      "/assets/entry.41ecdce4.js"
[GIN] 2022/11/10 - 11:39:13 | 200 |     145.676µs |   192.168.1.210 | GET      "/api/public/settings"
[GIN] 2022/11/10 - 11:39:13 | 200 |     137.378µs |   192.168.1.210 | GET      "/assets/logo.svg"
[GIN] 2022/11/10 - 11:39:13 | 200 |    1.081029ms |   192.168.1.210 | GET      "/123PAN/page/2"
[GIN] 2022/11/10 - 11:39:13 | 200 |     283.126µs |   192.168.1.210 | GET      "/api/me"
[GIN] 2022/11/10 - 11:39:13 | 200 |     115.641µs |   192.168.1.210 | GET      "/assets/useTitle.07ecabdb.js"
[GIN] 2022/11/10 - 11:39:13 | 200 |      81.941µs |   192.168.1.210 | GET      "/assets/useT.fc5c14e1.js"
[GIN] 2022/11/10 - 11:39:13 | 200 |     5.16224ms |   192.168.1.210 | GET      "/assets/Layout.fcfbc0e8.js"
[GIN] 2022/11/10 - 11:39:13 | 200 |     138.776µs |   192.168.1.210 | GET      "/assets/api.15bf4f8b.js"
[GIN] 2022/11/10 - 11:39:13 | 200 |     112.786µs |   192.168.1.210 | GET      "/assets/useUtil.2596da07.js"
[GIN] 2022/11/10 - 11:39:13 | 200 |   20.385367ms |   192.168.1.210 | GET      "/assets/Markdown.9c9fcc86.js"
[GIN] 2022/11/10 - 11:39:13 | 200 |    1.969329ms |   192.168.1.210 | GET      "/assets/index.a2a99c3e.js"
[GIN] 2022/11/10 - 11:39:13 | 200 |      46.648µs |   192.168.1.210 | GET      "/assets/index.11c046a7.js"
[GIN] 2022/11/10 - 11:39:13 | 200 |     104.906µs |   192.168.1.210 | GET      "/assets/FolderTree.a679b49c.js"
[GIN] 2022/11/10 - 11:39:13 | 200 |     116.053µs |   192.168.1.210 | GET      "/assets/logo.svg"
[GIN] 2022/11/10 - 11:39:13 | 200 |    1.133323ms |   192.168.1.210 | POST     "/api/fs/get"
[31mERRO[0m[2022-11-10 11:39:13] json: cannot unmarshal string into Go struct field Files.code of type int
failed to list objs
github.com/alist-org/alist/v3/internal/op.List.func1
	github.com/alist-org/alist/v3/internal/op/fs.go:59
github.com/alist-org/alist/v3/pkg/singleflight.(*Group[...]).doCall.func2
	github.com/alist-org/alist/v3/pkg/singleflight/singleflight.go:193
github.com/alist-org/alist/v3/pkg/singleflight.(*Group[...]).doCall
	github.com/alist-org/alist/v3/pkg/singleflight/singleflight.go:195
github.com/alist-org/alist/v3/pkg/singleflight.(*Group[...]).Do
	github.com/alist-org/alist/v3/pkg/singleflight/singleflight.go:108
github.com/alist-org/alist/v3/internal/op.List
	github.com/alist-org/alist/v3/internal/op/fs.go:56
github.com/alist-org/alist/v3/internal/fs.list
	github.com/alist-org/alist/v3/internal/fs/list.go:27
github.com/alist-org/alist/v3/internal/fs.List
	github.com/alist-org/alist/v3/internal/fs/fs.go:17
github.com/alist-org/alist/v3/server/handles.FsList
	github.com/alist-org/alist/v3/server/handles/fsread.go:77
github.com/gin-gonic/gin.(*Context).Next
	github.com/gin-gonic/gin@v1.8.1/context.go:173
github.com/alist-org/alist/v3/server/middlewares.Auth
	github.com/alist-org/alist/v3/server/middlewares/auth.go:55
github.com/gin-gonic/gin.(*Context).Next
	github.com/gin-gonic/gin@v1.8.1/context.go:173
github.com/alist-org/alist/v3/server/middlewares.StoragesLoaded
	github.com/alist-org/alist/v3/server/middlewares/check.go:11
github.com/gin-gonic/gin.(*Context).Next
	github.com/gin-gonic/gin@v1.8.1/context.go:173
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1
	github.com/gin-gonic/gin@v1.8.1/recovery.go:101
github.com/gin-gonic/gin.(*Context).Next
	github.com/gin-gonic/gin@v1.8.1/context.go:173
github.com/gin-gonic/gin.LoggerWithConfig.func1
	github.com/gin-gonic/gin@v1.8.1/logger.go:240
github.com/gin-gonic/gin.(*Context).Next
	github.com/gin-gonic/gin@v1.8.1/context.go:173
github.com/gin-gonic/gin.(*Engine).handleHTTPRequest
	github.com/gin-gonic/gin@v1.8.1/gin.go:616
github.com/gin-gonic/gin.(*Engine).ServeHTTP
	github.com/gin-gonic/gin@v1.8.1/gin.go:572
net/http.serverHandler.ServeHTTP
	net/http/server.go:2947
net/http.initALPNRequest.ServeHTTP
	net/http/server.go:3556
net/http.(*http2serverConn).runHandler
	net/http/h2_bundle.go:5911
runtime.goexit
	runtime/asm_amd64.s:1594 
[31mERRO[0m[2022-11-10 11:39:13] failed list /123PAN: json: cannot unmarshal string into Go struct field Files.code of type int
failed to list objs
github.com/alist-org/alist/v3/internal/op.List.func1
	github.com/alist-org/alist/v3/internal/op/fs.go:59
github.com/alist-org/alist/v3/pkg/singleflight.(*Group[...]).doCall.func2
	github.com/alist-org/alist/v3/pkg/singleflight/singleflight.go:193
github.com/alist-org/alist/v3/pkg/singleflight.(*Group[...]).doCall
	github.com/alist-org/alist/v3/pkg/singleflight/singleflight.go:195
github.com/alist-org/alist/v3/pkg/singleflight.(*Group[...]).Do
	github.com/alist-org/alist/v3/pkg/singleflight/singleflight.go:108
github.com/alist-org/alist/v3/internal/op.List
	github.com/alist-org/alist/v3/internal/op/fs.go:56
github.com/alist-org/alist/v3/internal/fs.list
	github.com/alist-org/alist/v3/internal/fs/list.go:27
github.com/alist-org/alist/v3/internal/fs.List
	github.com/alist-org/alist/v3/internal/fs/fs.go:17
github.com/alist-org/alist/v3/server/handles.FsList
	github.com/alist-org/alist/v3/server/handles/fsread.go:77
github.com/gin-gonic/gin.(*Context).Next
	github.com/gin-gonic/gin@v1.8.1/context.go:173
github.com/alist-org/alist/v3/server/middlewares.Auth
	github.com/alist-org/alist/v3/server/middlewares/auth.go:55
github.com/gin-gonic/gin.(*Context).Next
	github.com/gin-gonic/gin@v1.8.1/context.go:173
github.com/alist-org/alist/v3/server/middlewares.StoragesLoaded
	github.com/alist-org/alist/v3/server/middlewares/check.go:11
github.com/gin-gonic/gin.(*Context).Next
	github.com/gin-gonic/gin@v1.8.1/context.go:173
github.com/gin-gonic/gin.CustomRecoveryWithWriter.func1
	github.com/gin-gonic/gin@v1.8.1/recovery.go:101
github.com/gin-gonic/gin.(*Context).Next
	github.com/gin-gonic/gin@v1.8.1/context.go:173
github.com/gin-gonic/gin.LoggerWithConfig.func1
	github.com/gin-gonic/gin@v1.8.1/logger.go:240
github.com/gin-gonic/gin.(*Context).Next
	github.com/gin-gonic/gin@v1.8.1/context.go:173
github.com/gin-gonic/gin.(*Engine).handleHTTPRequest
	github.com/gin-gonic/gin@v1.8.1/gin.go:616
github.com/gin-gonic/gin.(*Engine).ServeHTTP
	github.com/gin-gonic/gin@v1.8.1/gin.go:572
net/http.serverHandler.ServeHTTP
	net/http/server.go:2947
net/http.initALPNRequest.ServeHTTP
	net/http/server.go:3556
net/http.(*http2serverConn).runHandler
	net/http/h2_bundle.go:5911
runtime.goexit
	runtime/asm_amd64.s:1594
failed get objs 
[GIN] 2022/11/10 - 11:39:13 | 200 |     49.3379ms |   192.168.1.210 | POST     "/api/fs/list"
[GIN] 2022/11/10 - 11:39:25 | 207 |    2.326291ms |   192.168.1.210 | PROPFIND  "/dav/"

**Comment by yoyo3287258** [comment]
一样，小量文件没问题，文件一大一多，就不行了
